cmd/version.go

```diff
@@ -20,11 +26,45 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
-	},
+	Run: showVersion,
+}
+
+// showVersion outputs the version of the exercism binary that is in us.
+func showVersion(cmb *cobra.Command, args []string) {
+	cmdName := RootCmd.Use
+	fmt.Printf("%s version %s", cmdName, Version)
+
+	if checkLatest {
+		c := cli.New(Version)
+
+		ok, err := c.IsUpToDate()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if ok {
+			fmt.Println(", is up to date.")
+			return
+		}
+
+		cv, _ := semver.Make(c.Version)
+		rv, _ := semver.Make(c.LatestRelease.Version())
+
+		switch {
+		case cv.LT(rv):
+			fmt.Println(", is out of date.")
+			fmt.Printf("To upgrade to %s, see `%s upgrade`.\n", rv, cmdName)
+		case cv.GT(rv):
+			fmt.Printf(", is newer than the latest version %s\n", rv)
```

Comment:
Let's assume that the only people with newer versions are people who have edited the CLI themselves... and it's OK if we tell them they're up to date, because the newest one hasn't been released yet.